src/main/Config.cpp

```diff
@@ -159,6 +159,7 @@ Config::Config() : NODE_SEED(SecretKey::random())
     CATCHUP_COMPLETE = false;
     CATCHUP_RECENT = 0;
     EXPERIMENTAL_PRECAUTION_DELAY_META = false;
+    EXPERIMENTAL_BACKGROUND_OVERLAY_PROCESSING = true;
```

Comment:
I think this should be false, no?

```suggestion
    EXPERIMENTAL_BACKGROUND_OVERLAY_PROCESSING = false;
```